test/pstl_offload/memory/allocation_utils.h

```diff
@@ -0,0 +1,127 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ALLOCATION_UTILS_H
+#define _ALLOCATION_UTILS_H
+
+#define _ONEDPL_PSTL_OFFLOAD_TOP_LEVEL
+#include <new>
+#include <cstdlib>
+#include <malloc.h>
+#include "support/utils.h"
+#undef _ONEDPL_PSTL_OFFLOAD_TOP_LEVEL
+
+extern "C"
+{
+void *__libc_malloc(std::size_t);
+void *__libc_calloc(std::size_t, std::size_t);
+void *__libc_memalign(std::size_t, std::size_t);
+void *__libc_realloc(void *, std::size_t);
+}
+
+#if __cpp_sized_deallocation
+#error "The test must be updated to cover a compiler with sized decallocation support."
+#endif
+
+#if TEST_DELETE_OVERLOAD
```

Comment:
I would suggest to rename this macro to `TEST_DELETE_SIZED_OVERLOAD`

Reply:
This macro was eliminated.